README.md

```diff
@@ -149,6 +149,13 @@ your `~/.emacs`:
 (setq utop-command "opam config exec -- utop -emacs")
 ```
 
+If you use `dune` and want to launch `dune utop` in emacs, you
+should add this to your `~/.emacs`:
+
+```scheme
```

Comment:
I believe "Emacs Lisp" is a valid language. We should check if that's correct and probably switch to it.

Reply:
I went ahead and changed the Scheme code blocks to Emacs Lisp (`elisp`). The highlighting seems to look correct on Github.